x-pack/solutions/search/plugins/search_solution/search_navigation/public/classic_navigation.ts

```diff
@@ -11,13 +11,13 @@ import type { CoreStart, ScopedHistory } from '@kbn/core/public';
 import type { ChromeNavLink, EuiSideNavItemTypeEnhanced } from '@kbn/core-chrome-browser';
 import type { SolutionNavProps } from '@kbn/shared-ux-page-solution-nav';
 
-import type { ClassicNavItem, ClassicNavItemDeepLink, ClassicNavigationFactoryFn } from './types';
+import type { ClassicNavItem, ClassicNavItemDeepLink } from './types';
 import { stripTrailingSlash } from './utils';
 
 type DeepLinksMap = Record<string, ChromeNavLink | undefined>;
 type SolutionNavItems = SolutionNavProps['items'];
 
-export const classicNavigationFactory: ClassicNavigationFactoryFn = (
```

Comment:
why did you remove this type?

Reply:
because its not used any where. No need to define a type if its no used.